src/components/admin/article/ArticleController.jsx

```diff
@@ -373,82 +336,50 @@ export default class ArticleController extends FalcorController {
             floatingLabelText="Title"
             fullWidth
           />
-          <form
-            onSubmit={this.handleSaveChanges}
-          >
-            <SelectField
-              floatingLabelText="Category"
-              maxHeight={400}
-              value={chosenCategory}
-              onChange={this.fieldUpdaters.category}
-              disabled={this.state.saving}
-              autoWidth={false}
-              style={{ width: 200 }}
-            >
-              {
-                _.map(categories, category => (
-                  <MenuItem
-                    value={category.slug}
-                    key={category.slug}
-                    primaryText={category.name}
-                  />
-                ))
-              }
-            </SelectField>
-            <TextField
-              name="image"
-              value={this.state.image}
-              floatingLabelText="Image (Remember to use https:// not http://)"
-              disabled={this.state.saving}
-              onChange={this.fieldUpdaters.image}
-              fullWidth
-            /><br />
-            <TextField
-              name="teaser"
-              floatingLabelText={
-                `Teaser (${this.state.teaser.length} of ${MAX_TEASER_LENGTH} characters)`
-              }
-              value={this.state.teaser}
-              disabled={this.state.saving}
-              onChange={this.fieldUpdaters.teaser}
-              multiLine
-              rows={2}
-              fullWidth
-            /><br />
-            <EditAuthorsForm
-              authors={this.state.authors}
-              onChange={this.debouncedHandleFormStateChanges}
-              handleAddAuthor={this.handleAddAuthor}
-              handleDeleteAuthor={this.handleDeleteAuthor}
-              model={this.props.model}
-              disabled={this.state.saving}
-            />
-            <RaisedButton
-              label={changedStateMessage}
-              primary
-              style={styles.buttons}
-              type="submit"
-              disabled={!this.state.changed || this.state.saving}
-            />
-          </form>
+          <CategorySelector
```

Comment:
And generalised here too :)

Reply:
Since I'm importing as default, I am going to use this name here - as it provides a bit more clarity in what this component manages. (I can do this as a className if people prefer though)